modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectRelationship/ObjectRelationship.tsx

```diff
@@ -49,7 +49,8 @@ function getLabel<T extends ObjectMap<any>>(item: T, key: keyof T) {
 
 	const label =
 		(value as LocalizedValue<string>)[defaultLanguageId] ??
-		(value as {[key: string]: string})['name'];
+		(value as {[key: string]: string})['name'] ?? 
+		(value as {[key: string]: string})['label'].toUpperCase();
```

Comment:
Hey @pedro-oliveira446, We can get the `label_i18n` instead of the `label`, the i18n label is already [internationalized](https://lingoport.com/what-is-i18n/), so it's better for our customers. 

Another point, if you look at the other relationship labels, none of them are capitalized, so I don't think we need `toUpperCase()` because the `label_i18n` is already capitalized

```ts
(value as {[key: string]: string})['label_i18n'];
```